docs/roles/RoleNORManager.md

```diff
@@ -70,7 +77,18 @@ In addition, the *Update dashboard run* action is automatically triggered on Mon
 ### Supporting the New Ontology requestor to pass the OBO Dashboard
 
 For many New Ontology requestors it is not always easy to interpret the warnings on the OBO Dashboard. The NOR Manager is responsible for supporting the requestor interpreting the dashboard (but _not_ to fix the problems). There is no need to dig deep into the ontologies to diagnose problems - that is the requestors responsibility - but pointing to documentation like the [OBO Foundry Principles](https://obofoundry.org/principles/fp-000-summary.html) may be necessary.
-- All discussions between the requestors and the NOR manager must take place in the [OBO Issue Tracker](https://github.com/OBOFoundry/OBOFoundry.github.io/issues) in the NOR issue created by the requestor.
+
+In addition, some checks need to be carried out manually by the NOR manager until sufficiently reliable automatic checks are available. A non exhaustive list includes:
+
+- URIs: check if it includes "https"  and if it conforms with the overall format "http://purl.obolibrary.org/obo/XXX"
+
+- Object properties: check if object properties are imported from the OBO-relations ontology. Check if domains and ranges are specified for created object properties.
```

Comment:
I dont think this should be done by the NOR, too difficult during a cursory review.

Reply:
```suggestion
```